### ISSUE TITLE
New updates on NA

### ISSUE THREAD

**Issue by Guowinson** [MathewSachin/Fate-Grand-Automata#142]
Hello guys, I found it quite disturbing since it wastes some time in total, which is why, there should be a feature where the app recognize a servants dying and press the screen to skip those few seconds. This especially comes in handy not only for arash user but also for future chen gong update

![Screenshot_2020-06-23-23-28-14-232_com aniplex fategrandorder en](https://user-images.githubusercontent.com/67333729/85431404-b5104980-b5ab-11ea-87b2-91f6d62abf88.jpg)

Yeah so this new update came. Do you think you could add a feature where the bot could choose whether or not to repeat this quest in the new update? Because my auto farm just stucks in there for hours, which needs me to keep pressing REPEAT QUEST, each battle, which kind beats the purpose of the app.

![Screenshot_2020-06-23-23-31-01-829_com aniplex fategrandorder en](https://user-images.githubusercontent.com/67333729/85430833-dcb2e200-b5aa-11ea-8398-d0decbd39f7f.jpg)

Also add a feature where we can set the bot to either send a friend request or ignore that notification please?

MOST IMPORTANTLY, 1 week ago, i tried downloading your app from the website from the link in your gamepress post. I tried multiple times, and the link runs but no downloads happening. A few days later i tried downloading stuff from playstore and the installation failed, where then i tried searching for answers in YouTube, and it tells me to :
Setting > turn on developer option > accessibility > developer option > turn off MIUI Optimization.
A few hours later, i remembered about your app and finally THE DOWNLOAD WORKS. A preview of my device available below, so yeah what I'm trying to say is, in case anyone is having the same problem, you could perhaps add this information in your downloads page. Because i almost gave up on your app, and thankfully i didn't.

![Screenshot_2020-06-23-23-53-33-432_com android settings](https://user-images.githubusercontent.com/67333729/85432207-d4f43d00-b5ac-11ea-8834-f53508324ecc.jpg)

All in all. THANKS A LOT FOR READING THIS SEMPAIS. AND I'M SORRY FOR REQUESTING A LOT OF STUFF. YOUR APPS HAS RETURN ME THE FREEDOM I LONG SINCE LOST THAT IS THE HOURS OF MY TIME FARMING EMBERS IN WHICH I COULD DO OTHER MORE PRODUCTIVE STUFF!

**Comment by MathewSachin** [comment]
### Death skip
To understand why this isn't realistically possible, you first need to understand how the app works. So, it has a few images stored in it which it matches with the things on the screen. So, we have images for buster card, arts card, attack button, etc...
Now, a servant dying doesn't have a common image and we can't have images for all servants dying in the app 😆. So, it should not be possible, but I'll be happy if it is proved otherwise.

### New Update
Of course, we're already working on it (#140).

### Friend requests
Friend requests should be ignored automatically by the app, but no guarantee since it is based on delays which can vary from time to time.

### MIUI Optimization
Never had that problem on my MIUI phone. Android 7, MIUI Global 11.0.2.
I just now noticed that `MIUI Optimization` is ON on my phone, but I have no problems. I think it should have been some other problem.

**Comment by Guowinson** [comment]
Hello again there, i checked ur site and saw V.14 came out. Tested it. My review :

1. Support choosing has significantly showed a difference. It's now able to identify the support immediately without any error in refreshing.
2. Auto skill now has faster input than the previous version. After inputting a skill, it then input another servant's skill without delay longer than the previous version. But, a skill which requires choosing a servant, Example (waver first skill) hasn't showed much difference, personally i don't mind since it's only at most 2 seconds difference then when input manually.
3. I use a 3T for farming embers, and in the last turn/battle, i don't need to input any skill, and just go straight for the NP. In the new version, the bot has around 3 seconds of delay before pressing the ATTACK button. Where in the previous version, there was jo delay in this part. I've only farmed less than 10 times. Perhaps because i re-download the whole app, and instead of updating it? Idk...
3. As mentioned in our previous interaction, you've finally add the REPEAT QUEST feature! Not to mention, in such a short amount of time. I NOW PRAY FOR YOUR ACQUISITION OF SKADI MORE VIVIDLY THAN EVER! That will be all from me, thanks again sempai! Feel free to close this thread after you read this hahaha.
